markdown/dt-otel/dt-otel.md

```diff
@@ -111,12 +111,95 @@ The `.env` file contains several environment variables that are used within the
 * Open Telemetry contribution repository collector docker image tag and version `otel/opentelemetry-collector-contrib:0.67.0`
 * Solace PubSub+ broker docker image tag and version `solace/solace-pubsub-standard:10.2`
 
-## Configuring the PubSub+ Event Broker: Software
+## Three Options for Config Management
+Duration: 0:03:00
+
+You can (generally) configure the Solace PubSub+ broker using three different methods; this section provides a very brief outline of those.
+
+Each section of this CodeLab that performs a configuration step on the Solace broker will include all three options.
+
+Negative
```

Comment:
The latest version of `claat` does not support `Negative/Positive : ` notations. We will have to change this to aside tags as follows

```
<aside class=“negative”> Do not perform all 3 configuration options, just choose 1 for each section </aside>
```